support-scripts/rebuild-rabbitmq.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+cd /opt/rpc-openstack/openstack-ansible/playbooks
+
+eval $@
```

Comment:
This is a bad idea. People construct command lines in all sorts of creative ways, and 99% of the time they aren't expecting arguments passed to a script to be executed.

This sort of thing looks inocuous but may catch someone out when they really don't need it (Eg customer down). 

I suggest removing L4 and replacing the test on L5 with ```[[ "$1" != "iknowwhatido" ]]```. L6 will need =1 removing.